code/game/objects/items/devices/radio/headset.dm

```diff
@@ -163,7 +163,7 @@ GLOBAL_LIST_EMPTY(secsets)
 	var/message = show_radial_menu(user, user, options)
 	if(!message)
 		return FALSE
-	talk_into(src, "Dispatch, [message] at [get_area(user)], requesting response.", radio_channel)
+	talk_into(src, "Dispatch, Officer [user], [message] at [get_area(user)], requesting response.", radio_channel)
```

Comment:
get_area needs to be genericized, probably to department.

Im unaware of any proc for that though, so if someone has an insight, lmk

Reply:
There is no current way to do that as far as I am aware

Reply:
Gotcha. Ill probably do a snowflake list with strings. It'll be ugly, but it will work for the testmerge